photon-lib/py/photonlibpy/multiTargetPNPResult.py

```diff
@@ -0,0 +1,45 @@
+from dataclasses import dataclass, field
+from wpimath.geometry import Transform3d
+from photonlibpy.packet import Packet
+
+
+@dataclass
+class PNPResult:
+    _NUM_BYTES_IN_FLOAT = 8
+    PACK_SIZE_BYTES = 1 + (_NUM_BYTES_IN_FLOAT * 7 * 2) + (_NUM_BYTES_IN_FLOAT * 3)
+
+    isPresent: bool = False
+    best: Transform3d = field(default_factory=Transform3d)
+    alt: Transform3d = field(default_factory=Transform3d)
```

Comment:
wpimath.geometry objects are immutable, so creating them immediately is safe.


```suggestion
    best: Transform3d = Transform3d()
    alt: Transform3d = Transform3d()
```

Reply:
Hmmm, I can try again I guess? I got errors when running the test locally because of mutability... but maybe I misread the error.

Reply:
![image](https://github.com/PhotonVision/photonvision/assets/4583662/b7cbe230-112e-4d05-8932-c8626989d031)
At least in my testcase.... python is complaining about them not being immutable?